src/jira/client/axios.ts

```diff
@@ -50,7 +54,18 @@ const getErrorMiddleware = (logger: Logger) =>
 	 * @param {import("axios").AxiosError} error - The error response from Axios
 	 * @returns {Promise<Error>} The rejected promise
 	 */
-	(error: AxiosError): Promise<Error> => {
+	(error: AxiosError): Promise<Error|SkipRedirect> => {
+		/**
+		 * EDGE CASE:
+		 * When sending a POST request to `https://OLD.atlassian.net/rest/deployments/0.1/bulk`, it sends a 302 response,
+		 * and redirects to a GET request `https://NEW.atlassian.net/rest/deployments/0.1/bulk`
+		 * This GET request is now failing with a 405 response, which is not an error on the app side
+		 * So treating this as a non-error case
+		 */
+		if (error.request.method === "GET" && error.request.path === "/rest/devinfo/0.10/bulk") {
```

Comment:
Nice, 👍 , and depends on the logs/result, we might want to expand the request.path to include other jira method paths as well if those have 405 as well. But for now, lets just start with that /devinfo/bulk first.